robot/Cumulus/resources/Data.py

```diff
@@ -0,0 +1,7 @@
+from tasks.npsp_delete import DeleteData
+from tasks.salesforce_robot_library_base import SalesforceRobotLibraryBase
+
+
+class Data(SalesforceRobotLibraryBase):
+    def delete(self, objects, *, where=None, hardDelete=False):
```

Comment:
I'd suggest calling this bulk_delete to help disambiguate from the salesforce_delete keyword in the Salesforce library, which deletes a single record.

Reply:
https://github.com/SalesforceFoundation/Cumulus/pull/4592/files#diff-cab735d807b27fbf170d4ed5b5685119R6